File: go/utility/make_context.go
package utility

import "github.com/voxgig-sdk/kokkai-kaigiroku-api-sdk/go/core"

func makeContextUtil(ctxmap map[string]any, basectx *core.Context) *core.Context {
	return core.NewContext(ctxmap, basectx)
}

File: .sdk/tm/py/pkg/utility/make_context.py
# KokkaiKaigirokuApi SDK utility: make_context

from projectname_sdk.core.context import KokkaiKaigirokuApiContext


def make_context_util(ctxmap, basectx):
    return KokkaiKaigirokuApiContext(ctxmap, basectx)
